src/lib/isLicensePlate.js

```diff
@@ -0,0 +1,106 @@
+import assertString from './util/assertString';
+
+const validators = {
+  'de-LI': str => /^FL[- ]?\d{1-5}[UZ]?$/.test(str),
+  'sq-AL': str => /^[A-Z]{2}[- ]?((\d{3}[- ]?(([A-Z]{2})|T))|(R[- ]?\d{3}))$/.test(str),
+  'de-DE': (str) => {
+    // Rule 1: License plate must match regex
+    const regex = /^[A-Z]{1,3}[- ]?[A-Z]{1,2}[- ]?[0-9]{1,4}$/;
+    const plateSections = str.split(/[ -]/g);
+    const validFormat = regex.test(str);
+    // Rule 2: First section (1-3 letters until sepatator) must be a valid county code
+    const countyCodes = [
+      'AW', 'AIC', 'FDB', 'UL', 'ABG', 'SLN', 'AK', 'SAW', 'GA', 'KLZ', 'AÖ',

```

Comment:
A regex to cover all these will be a little complex but can we try as opposed to listing manually?

Reply:
Can do that!